net/modemmanager/files/modemmanager.proto

```diff
@@ -392,6 +395,12 @@ proto_modemmanager_setup() {
 		cliauth="${cliauth}${cliauth:+|}$auth"
 	done
 
+	# set logging level
+	[ -n "${loglevel}" ] && {
+		echo "Setting logging level to: ${loglevel}"
+		mmcli -G ${loglevel}
```

Comment:
As said earlier, this won't work properly as soon as you have more than one modem in the system. The daemon log level is a daemon option that applies to all modems.